docs/interactions.rst

```diff
@@ -0,0 +1,142 @@
+:orphan:
+
+.. currentmodule:: nextcord
+
+.. _client_index:
+   
+Interaction Commands
+======================
+
+Since Discord has added interaction commands a feature with many possibilities, we at Nextcord have decided to add them to our fleet of features.
+
+This doc will explain the innerworkings and how to use interaction commmands.
+
+We suggest you learn how to make regular commands before looking through here.
+
+How To Make A Simple Interaction Command
+------------------------------------------
+
+This right here is a simple ping pong Command made with Nextcords slash feature.
+
+.. code-block:: python3
+
+    @bot.slash_command(name="ping")
+    async def ping(interaction):
+        await interaction.response.send_message("Pong!")
+        
+The way it works is that you use the ``slash_command`` function to interact with the Discord API. The ``name`` parameter is the name of your slash command.
```

Comment:
``slash_command`` should be a cross link to it's docs

Reply:
currently i haven't been able to get the api docs to work so i will do this when i do